python-lib/ontology_tagging_formatting.py

```diff
@@ -1,7 +1,3 @@
-"""
-Module to write the output dataframe depending on the output format
```

Comment:
Why remove that?

Reply:
I didn't, I moved it under the class declaration